Source/Venice/Channel/Channel.swift

```diff
@@ -67,26 +67,23 @@ public final class Channel<T>: Sequence {
     }
 
     /// Closes the channel. When a channel is closed it cannot receive values anymore.
-    public func close() -> Bool {
-        if closed {
-            return false
-        }
+    public func close() throws {
+        guard !closed else { return }
```

Comment:
i don't think it should be Closable, but should have similar api instead, the only question is do we need `guard !closed else { throw AlreadyClosed() }` or just `return`.